### PR TITLE
Rewrite doconv and plone instance sections.

### PR DESCRIPTION
**Pull request by allcaps**

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by gyst** [comment]
Jenkins test this please

File: docs/installation/development.rst [hunk 1]
```diff
@@ -20,10 +20,6 @@ Make sure you have the OS-level packages you need to build Plone, this can be
 achieved using `install.plone.dependencies`_.
 
 In addition, for the Plone Intranet file preview functionality, you need docsplit.
-On Ubuntu::
-
-    sudo apt-get install ruby
-    gem install docsplit
```
Comment (gyst) [inline docs/installation/development.rst:26]: Please don't remove the linux install documentation for this.

**Comment by allcaps** [comment]
Jenkins test this please Please listen to mee too! ;)

**Pull request merged**